### ISSUE TITLE
Unknown array in BatchNormalization

### ISSUE THREAD

**Issue by PABCSoft** [tensorlayer/tensorlayer#898]
### New Issue Checklist

-  I have read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
-  I searched for [existing GitHub issues](https://github.com/tensorlayer/tensorlayer/issues)

### Issue Description
When you save the layer data settings to .npz, arrays of the following sizes are saved:
(3, 3, 64, 64)
(64)
(64)
(64)
(64)
What does the fifth array contain?

### Reproducible Code

OS: Windows7

```
nn = Conv2d(n, 64, (3, 3), (1, 1), act=None, padding='SAME', W_init=w_init, b_init=b_init, name='n64s1/c1/%s' % i)
            nn = BatchNormLayer(nn, act=tf.nn.relu, is_train=is_train, gamma_init=g_init, name='n64s1/b1/%s' % i)
```
